web_widget_model_viewer/static/src/js/web_widget_model_viewer.js

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Lorenzo Battistini @ TAKOBI
+// Copyright 2020 Andrea Piovesana @ Openindustry.it
+odoo.define('web_widget_model_viewer.FieldBinaryModelViewer', function (require) {
+    "use strict";
+
+    var BasicFields = require('web.basic_fields');
+    var core = require('web.core');
+    var registry = require('web.field_registry');
+    var session = require('web.session');
+    var utils = require('web.utils');
+    var _t = core._t;
+    var qweb = core.qweb;
+
+    var FieldBinaryModelViewer = BasicFields.FieldBinaryFile.extend({
+        template: 'FieldBinaryModelViewer',
+        events: _.extend({}, BasicFields.FieldBinaryFile.prototype.events, {
+            'click': function () {
+                if (this.mode === "readonly") {
+                    this.trigger_up('bounce_edit');
+                }
+            },
+            'click #model-viewer-fullscreen': 'fullscreen',
+        }),
+        supportedFieldTypes: ['binary'],
+        init: function () {
+            this._super.apply(this, arguments);
+            this.max_upload_size = 250 * 1024 * 1024; // 250M
```

Comment:
Would be nice if you can make it customizable instead of hard coded

Reply:
Right, done: https://github.com/OCA/web/compare/97964364b8b56da2d8dd3a7cbc622f126007835d..287a427735482351eb60d190385a693018758d06

Reply:
Very good! Thanks!